ui/pages/swaps/build-quote/build-quote.js

```diff
@@ -627,44 +627,38 @@ export default function BuildQuote({
                   alt={t('swapSwapSwitch')}
                 />
               </Box>
-              <Typography variant={TypographyVariant.H7} marginTop={0}>
+              <Text variant={TextVariant.bodySm} as="h6" marginTop={0}>
                 {t('stxDescription')}
-              </Typography>
-              <Typography
+              </Text>
+              <Text
                 as="ul"
-                variant={TypographyVariant.H7}
-                fontWeight={FONT_WEIGHT.BOLD}
+                variant={TextVariant.bodySm}
+                fontWeight={FontWeight.Bold}
                 marginTop={3}
               >
                 <li>{t('stxBenefit1')}</li>
                 <li>{t('stxBenefit2')}</li>
                 <li>{t('stxBenefit3')}</li>
                 <li>
                   {t('stxBenefit4')}
-                  <Typography
+                  <Text
                     as="span"
-                    fontWeight={FONT_WEIGHT.NORMAL}
-                    variant={TypographyVariant.H7}
+                    fontWeight={FontWeight.Normal}
+                    variant={TextVariant.bodySm}
                   >
                     {' *'}
-                  </Typography>
+                  </Text>
                 </li>
-              </Typography>
-              <Typography
-                variant={TypographyVariant.H8}
+              </Text>
+              <Text
+                variant={TextVariant.bodyXs}
+                as="h6"
                 color={TextColor.textAlternative}
                 boxProps={{ marginTop: 3 }}
               >
                 {t('stxSubDescription')}&nbsp;
-                <Typography
-                  as="span"
-                  fontWeight={FONT_WEIGHT.BOLD}
-                  variant={TypographyVariant.H8}
-                  color={TextColor.textAlternative}
-                >
-                  {t('stxYouCanOptOut')}&nbsp;
-                </Typography>
-              </Typography>
+                <strong>{t('stxYouCanOptOut')}&nbsp;</strong>
```

Comment:
I think it would be better to use the fontWeight prop instead of the strong tag here
```suggestion
                {t('stxYouCanOptOut')}&nbsp;
```